tasks/handle_container_conf_d.yml

```diff
@@ -0,0 +1,35 @@
+---
+
+- name: Ensure containers.d exists - system
+  file:
+    path: "{{ __podman_containers_conf_system|dirname }}"
```

Comment:
Something weird is happening with this macro eval - there is a double quote being added to the path!
```
    "path": "\"/etc/containers/containers.conf.d",
```
which I think is why the test fails - see also
```
    "dest": "\"/etc/containers/containers.conf.d/50-systemroles.conf\"\n",
```
which has extra leading and trailing double quotes (and newline?) - maybe the `dirname` filter is errant?

Reply:
oh goodness ... that's fun, when I run the tox tests locally it worked fine on Fedora 36 ... alright, I'll dig into that a bit. Thanks!